setup.py

```diff
@@ -3,6 +3,7 @@
 # To use a consistent encoding
 from codecs import open
 from os import path
+import frag_pele as fp
```

Comment:
To do so we should change the __version__ in __init__.py

Reply:
Done! I forgot to push it